README.md

```diff
@@ -725,6 +726,38 @@ $middlewares = new MiddlewareRunner([
 ]);
 ```
 
+#### RequestBodyParserMiddleware
+
+Another built-in middleware is `RequestBodyParserMiddleware` which takes a fully buffered request body 
```

Comment:
I think this should read just

> The `RequestBodyParserMiddleware` takes a fully buffered request body...

Referring to previous paragraphs and repeating "built-in middleware..." might be become tedious once we add more and more middleware.